.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Documentation issue
+about: Report a problem or suggest changes for the documentation at https://typing.readthedocs.io/
+title: ''
```

Comment:
There's no need to add `title: ''`

Reply:
This was added automatically by the templates generator.